lib/ramble/ramble/object_mixin.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022-2025 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.util import format
+
+
+class ObjectMixin:
```

Comment:
Could we move this to be a base_class in the builtin repo? That way it's easy to extend the mixin.